src/mame/drivers/oric.cpp

```diff
@@ -957,6 +960,8 @@ ROM_START(orica)
 	ROMX_LOAD( "bas122sw_le.rom", 0, 0x04000, CRC(e7fd57a4) SHA1(c75cbf7cfafaa02712dc7ca2f972220aef86fb8d), ROM_BIOS(22))
 	ROM_SYSTEM_BIOS( 23, "ver122uk", "Basic 1.22 UK")
 	ROMX_LOAD( "bas122uk_le.rom", 0, 0x04000, CRC(9865bcd7) SHA1(2a92e2d119463e682bf10647e3880e26656d65b5), ROM_BIOS(23))
+	ROM_SYSTEM_BIOS( 24, "ver10uk", "Basic 1.0 UK")
+	ROMX_LOAD( "basic10.rom", 0, 0x04000, CRC(d6006a01) SHA1(84759cc8faae53070f1c8ce8408982a0edcb0796), ROM_BIOS(24))
```

Comment:
If it’s not for the Atmos, it shouldn’t be here.

Reply:
Thanks for catching this, I forgot to delete the ROM from the Atmos side after moving it to the -1
I didn't include a readme.txt with this submission - credit goes to "tante ju" who's active in the German VzEkC forum.